nero_view.py

```diff
@@ -1,12 +1,9 @@
 import customtkinter
 from tkinter import Frame
+from typing import List
 from modes.debug_mode.debug_mode import DebugMode
-from modes.efficiency_mode.efficiency_mode import EfficiencyMode
-from modes.off_mode.off_mode import OffMode
-from modes.charging_mode.charging_mode import ChargingMode
-from modes.pit_lane_mode.pit_lane_mode import PitLaneMode
-from modes.reverse_mode.reverse_mode import ReverseMode
-from modes.speed_mode.speed_mode import SpeedMode
+from modes.mode import Mode
+from models.mock_model import MODES
```

Comment:
the mock model should not be the source of truth for the list of modes

that should probably be here or in some file dedicated to constants if we want